test/helpers/db.js

```diff
@@ -9,18 +9,10 @@ const Notification = require('../../src/models/db/notification').Notification
 const Fulfillment = require('../../src/models/db/conditionFulfillment').ConditionFulfillment
 
 exports.init = function * () {
+  yield knex.migrate.rollback(knexConfig).then()
```

Comment:
I'm still getting the indices errors even with this rollback. Looks like a knex issue: https://github.com/tgriesser/knex/issues/729